libs/net/dal/Services/ReportInstanceService.cs

```diff
@@ -161,6 +161,13 @@ public ReportInstance Update(ReportInstance entity, bool updateChildren = false)
                 var originalInstanceContent = originalInstanceContents.FirstOrDefault(o => o.ContentId == ric.ContentId && o.SectionName == ric.SectionName);
                 if (originalInstanceContent == null)
                 {
+                    // Fetch the original content from the database to compare.
+                    originalContent = this.Context.Contents
```

Comment:
This probably won't happen in production, but locally the Elasticsearch data is not always synced with the database.